web/js/palettes/legend.js

```diff
@@ -118,11 +118,13 @@ export function palettesLegend(spec) {
   var updateClasses = function (legend, index) {
     var $panel = $(selector + ' [data-index=\'' + index + '\']');
     $panel.empty();
+    var $detailPanel = $('<div></div>');
     loEach(legend.colors, function (color, classIndex) {
-      var $colorBox;
+      var label = legend.tooltips[classIndex];
+      label = (legend.units) ? label + ' ' + legend.units : label;
       var $runningDataPointLabel = $('<span></span>')
         .addClass('wv-running-category-label');
-      $colorBox = $('<span></span>')
+      var $colorBox = $('<span></span>')
```

Comment:
It looks like the `$colorBox` in the first loop and the `$colorBox` in the second loop are different HTML elements, so I think they should use two different variables. You could either give the two elements different variable names, which is probably easiest, or refactor a bit more and eliminate the need to store the "color box" in an intermediate variable at all. What do you think?